cache/cache.h

```diff
@@ -5,6 +5,8 @@
 #include <stdlib.h>
 #include <string.h>
 
+#include "types/types.h"
```

Comment:
Is this inclusion necessary?

Reply:
For defined macros used in backeend_cache.c

Reply:
Affirmative. However, I would like to change the name scheme `"types/types.h"`, which looks lousy. I'll propose changes later.